modules/cli/pages/cli-intro.adoc

```diff
@@ -34,21 +34,26 @@ You can collect diagnostics through the command-line interface by using either t
 
 For convenience, the following EE Server utilities are also available in a tools package that you can download --
 xref:tools:cbimport.adoc[cbimport], xref:tools:cbexport.adoc[cbexport], and xref:backup-restore:cbbackupmgr.adoc[cbbackupmgr].
+This allows developers, testers, and others to use the tools from machines on which Couchbase Server is not installed.
 
 Download the command line tools package for your platform through the following links:
 
-* https://packages.couchbase.com/releases/7.1.3/couchbase-server-tools_7.1.3-linux_x86_64.tar.gz[https://packages.couchbase.com/releases/7.1.3/couchbase-server-tools_7.1.3-linux_x86_64.tar.gz]
-* https://packages.couchbase.com/releases/7.1.3/couchbase-server-tools_7.1.3-linux_aarch64.tar.gz[https://packages.couchbase.com/releases/7.1.3/couchbase-server-tools_7.1.3-linux_aarch64.tar.gz]
-* https://packages.couchbase.com/releases/7.1.3/couchbase-server-tools_7.1.3-macos_x86_64.zip[https://packages.couchbase.com/releases/7.1.3/couchbase-server-tools_7.1.3-macos_x86_64.zip]
-* https://packages.couchbase.com/releases/7.1.3/couchbase-server-tools_7.1.3-windows_amd64.zip[https://packages.couchbase.com/releases/7.1.3/couchbase-server-tools_7.1.3-windows_amd64.zip]
+
+* https://packages.couchbase.com/releases/7.2.0/couchbase-server-tools_7.2.0-linux_x86_64.tar.gz[https://packages.couchbase.com/releases/7.2.0/couchbase-server-tools_7.2.0-linux_x86_64.tar.gz]
+
+* https://packages.couchbase.com/releases/7.2.0/couchbase-server-tools_7.2.0-linux_aarch64.tar.gz[https://packages.couchbase.com/releases/7.2.0/couchbase-server-tools_7.2.0-linux_aarch64.tar.gz]
+
+* https://packages.couchbase.com/releases/7.2.0/couchbase-server-tools_7.2.0-macos_x86_64.zip[https://packages.couchbase.com/releases/7.2.0/couchbase-server-tools_7.2.0-macos_x86_64.zip]
+
```

Comment:
There is now a macos_arm64 build -- so need to add this as well:
https://packages.couchbase.com/releases/7.2.0/couchbase-server-tools_7.2.0-macos_arm64.zip